scrapy/utils/reactor.py

```diff
@@ -50,13 +50,19 @@ def __call__(self):
         return self._func(*self._a, **self._kw)
 
 
-def install_reactor(reactor_path):
+def install_reactor(reactor_path, event_loop_path=None):
     """Installs the :mod:`~twisted.internet.reactor` with the specified
-    import path."""
+    import path. Also installs the asyncio event loop with the specified import
+    path if the asyncio reactor is enabled"""
     reactor_class = load_object(reactor_path)
     if reactor_class is asyncioreactor.AsyncioSelectorReactor:
         with suppress(error.ReactorAlreadyInstalledError):
-            asyncioreactor.install(asyncio.get_event_loop())
+            if event_loop_path is not None:
+                event_loop_class = load_object(event_loop_path)
+                event_loop = event_loop_class()
+            else:
+                event_loop = asyncio.new_event_loop()
```

Comment:
Is there any reason to use `new_event_loop` here as opposed to `get_event_loop` as before?

Reply:
Calling `get_event_loop` may set the asyncio event loop globally since it calls  `set_event_loop`(which sets the loop as the default event loop)  along with `new_event_loop`(which creates a new event loop object). If someone plans to use some different asyncio event loop for work other than web crawling using scrapy in the same project, it may lead to issues.

Reply:
This was discussed earlier so I tried to avoid using `set_event_loop`. https://github.com/scrapy/scrapy/pull/4414#discussion_r398017430